src/js/sdk.tsx

```diff
@@ -106,6 +108,7 @@ export function init(passedOptions: ReactNativeOptions): void {
       ),
     ]);
 
+    defaultIntegrations.push(new NativeLinkedErrors());
```

Comment:
What's about https://github.com/getsentry/sentry-react-native/issues/2118
Does this PR help with that?

Reply:
Thanks for mentioning this error, sadly no, in debug only the top-level js error would be symbolicated.